alanbeals/grpfinallshopping
# Db/InitialWork - Create initial DB work

### What's changed:
- Create tables
- CRUD for user, orderItem, order, products (not all of them have full CRUD)

### How to test it:
- I've left some commented code on MainActivity. Uncomment each line one by one and run the project.
- Verify that the grocery_db was created on the Device File Explorer
- Do what we did during class -> Save the DB file, open it with SQLite and check that the tables were populated accordingly.
